README.md

```diff
@@ -1 +1,241 @@
+<br>
+<!-- > Describe how a new developer could make use of your project.
+To get a local copy up and running, follow these steps. -->
+
+<!-- ### Prerequisites -->
+
+<!-- > In order to run this project you need: -->
+
+<!--
+Example command:
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+> Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/amarmahdi1112/portfolio-amar-muha.git
+```
+<!--
+Example commands:
+--->
+
+<!-- ### Install
+
+Install this project with: -->
+
+<!--
+Example command:
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project:
+
+```sh
+  Go to '[root-directory]/my-folder/portfolio-amar-muha/main/' folder and execute the 'index.html' file
+
+```
+  > You can use the [live](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) server from vscode to run the index file 
+<!--
+Example command:
+--->
+
+<!-- ### Run tests
+
+To run tests, run the following command: -->
+
+<!--
+Example command:
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+<!-- ### Deployment
+
+You can deploy this project using: -->
+
+<!--
+Example:
+```sh
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## Author <a name="authors"></a>
+
+<!-- > Mention all of the collaborators of this project. -->
+Amar Muha
+- GitHub: [@amarmahdi1112](https://github.com/amarmahdi1112)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/amar-muha-698080178)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+<!-- > Describe 1 - 3 features you will add to the project. -->
+- **Desktop Responsive Design**
+<!-- - [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]** -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+<!-- ## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- SUPPORT -->
+
+<!-- ## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- ACKNOWLEDGEMENTS -->
+
+<!-- ## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- FAQ (optional) -->
+
+<!-- ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- LICENSE -->
```

Comment:
- [ ] This `faq` section is the only optional section of your readme file, make sure to remove it rather than commenting it out. commented code is not clean code and it is unprofessional please make sure to kindly fix this before making your next review request :+1: